packages/admin/blocks-admin/src/blocks/factories/createCompositeBlock.tsx

```diff
@@ -64,7 +64,9 @@ type PickType<T, K extends AllBlockKeys<T>> = T extends { [k in K]?: any } ? T[K
 type ExtractBlockConfig<T, K extends string | number | symbol> = K extends AllBlockKeys<T> ? PickType<T, K> : never;
 
 type ExtractCompositeBlocksConfig<Options extends CreateCompositeBlockOptions> = Options extends CreateCompositeBlockOptionsWithGroups
-    ? ExtractCompositeBlocksConfigBase<MergeBlockConfigs<ExtractGroupConfigs<Options["groups"]>["blocks"]>>
+    ? // @ts-expect-error MergeBlockConfigs somehow results in type Record<string, {}> here, which doesn't satisfy constraint
```

Comment:
@nsams if you know how to fix this, be my guest. @rabengraph would know it for sure 😁

Reply:
😄  @ts-expect-error is just fine!

Reply:
Nevermind, I've fixed it somehow 😅